packages/create-sitecore-jss/src/common/args/base.ts

```diff
@@ -0,0 +1,17 @@
+import { ClientAppAnswer } from '../prompts/base';
+
+type Arg = string | number | boolean;
+
+export interface BaseArgs {
```

Comment:
It's a base set of arguments, which we don't prompt.
Basically
Args - flags which we can provide (questions will be skipped)
Answers - exact types which we should handle in case if user answered

So as you can see, args contain part of Answers interface, and all properties are optional (destination we set up at the start), because it's not required for user to provide all flags, but he can do it